src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewController.swift

```diff
@@ -293,25 +293,32 @@ private extension UITableViewCell {
 
 	func addSeparator(_ location: SeparatorLocation) {
 		let separator = UIView(frame: bounds)
-		contentView.addSubview(separator)
+		
+		let backgroundView: UIView
+		if let existingBackgroundView = self.backgroundView {
+			backgroundView = existingBackgroundView
+		} else {
+			backgroundView = UIView()
+			self.backgroundView = backgroundView
+		}
+		backgroundView.addSubview(separator)
+		
 		separator.backgroundColor = .enaColor(for: .hairline)
 		separator.translatesAutoresizingMaskIntoConstraints = false
-		separator.trailingAnchor.constraint(equalTo: contentView.layoutMarginsGuide.trailingAnchor).isActive = true
+		separator.leadingAnchor.constraint(equalTo: backgroundView.leadingAnchor, constant: 16).isActive = true
+		separator.trailingAnchor.constraint(equalTo: backgroundView.trailingAnchor, constant: -16).isActive = true
```

Comment:
```suggestion
		separator.leadingAnchor.constraint(equalTo: contentView.layoutMarginsGuide.leadingAnchor, constant: -2.0).isActive = true
		separator.trailingAnchor.constraint(equalTo: contentView.layoutMarginsGuide.trailingAnchor, constant: 32.0).isActive = true
```
Use still contentView for layout

Reply:
revert it.